docs/taskruns.md

```diff
@@ -334,6 +334,10 @@ If multiple `Steps` are defined in the `Task` invoked by the `TaskRun`, you can
 status in the `steps.results` field using the following command, where `<name>` is the name of the target
```

Comment:
thanks @psschwei 

since we are changing this section, can I please ask you to correct one more thing?

please replace `steps.results` with `status.steps`. 

Tekton stores status under `status.steps`:

```
status:
  steps:
  - container:
    imageID:
    name:
    terminated:
      containerID:
      exitCode:
```

Reply:
done

Reply:
Thanks @psschwei 😸